lib/universal-search.js

```diff
@@ -173,6 +180,7 @@ Search.prototype = {
     Cu.unload('chrome://universalsearch-ui/content/urlbar.js', win.universalSearch);
 
     Cu.unload('chrome://universalsearch-lib/content/recommendation-server.js', win.universalSearch);
+    Cu.unload('chrome://universalsearch-lib/content/logger.js', win.universalSearch);
```

Comment:
Per the comments a few lines down, you can't unload the logger until the last call is made in the unloadFromWindow method, or this will break uninstall